python/metaparticle_pkg/docker_builder.py

```diff
@@ -0,0 +1,8 @@
+import subprocess
+
+class DockerBuilder:
+    def build(self, img):
+        subprocess.call(['docker', 'build', '-t', img, '.'])
```

Comment:
I’m really looking forward to metaparticle for Python! You said comments welcome on twitter so here we go. 

subprocess also has check_call that will raise if the process exits with an error code. As the call isn’t returned here, exceptions could be the only way users find out there’s a problem. Exceptions are normal for Python developers too.